engineyard/ey-cookbooks-stable-v5
# Passenger5 cleanup

#### Description of your patch
Moving nginx configuration from passenger5 recipe to nginx recipe **when passenger5 is selected**

#### Recommended Release Notes
Enable HTTP/2 on Passenger5

#### Estimated risk
HIGH

#### Components involved
nginx & passenger5

#### Description of testing done
See QA instructions

#### QA Instructions
Boot V5 selecting passenger5. Use the following variants:

1. With an SSL cert
2. With "http2" custom recipe enabled 
3. Without SSL cert 

Verify that passenger5/nginx configuration is identical against a vanilla passenger5 environment. 

**Files affected:**

- /etc/nginx/servers/#APP#.conf
- /etc/nginx/servers/#APP#.ssl.conf
- /etc/nginx/common/proxy.conf
